### ISSUE TITLE
A request if you may adhere

### ISSUE THREAD

**Issue by muhammadanas0716** [kannanjayachandran/Wafers-Fault-Detection#4]
I have a repo on GitHub with **99** stars as of now and it's based on ML Projects,

I'd love it if you add this amazing project of yours to my repo.

Link: https://github.com/muhammadanas0716/Machine-Learning-101

**Comment by muhammadanas0716** [comment]
Absolutely! Just keep updating the project on the repo as you add more stuff!

I'd love to have a look at your PR. And thank you for your appreciation!

**Comment by kannanjayachandran** [comment]
Ok. Do I need to replicate the entire project there or provide a jupyter notebook and data; as other project in your repository contains a notebook and the data used.

**Comment by muhammadanas0716** [comment]
Have a look at how I have structured my projects. Just like that.
